viewer/packages/camera-manager/src/Keyboard.ts

```diff
@@ -32,8 +32,12 @@ const EVENT_CODES = [
 
 type EventCode = (typeof EVENT_CODES)[number];
 
+const isEventCode = (value: string): value is EventCode => {
+  return EVENT_CODES.some(eventCode => eventCode === value);
```

Comment:
Can we put the events in a set as a pre-processing step to make this lookup faster?

Reply:
Done in: [140c08c](https://github.com/cognitedata/reveal/pull/3620/commits/140c08cd96b3f1b6d81a834ed03992cdd9ccb551)